### PR TITLE
feat: add vite-electron-builder

### PR DESCRIPTION
**Pull request by cawa-93**
- [x] Pull request template structure not broken

### `Type`

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [x] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md).

> 👉  _Put an `x` in the boxes that apply._

- [x] Title as described
- [x] Make sure you put things in the right category!
- [x] Always add your items to the end of a list

#### `Open Source`

- [x] Link description does not contain a link to an author / third-party resource
- [x] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product

#### `Apps/Websites`

- [ ] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using Vite intensively.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.
- [ ] A commercial product using Vite, provided that guests could reasonably check out how Vite was used A commercial product using Vite, provided that guests could reasonably check out how Vite was used (i.e. A headless CMS which uses Vite for the Admin/editor Area and offers a free tier).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vue-vben-admin-2.0](https://github.com/anncwb/vue-vben-admin) - Starter template with Vue 3, TypeScript, Tailwind CSS, Ant Design Vue2.
 - [electron-vue-next](https://github.com/ci010/electron-vue-next) - Starter template using Vue 3 in Vite and latest Electron with fluent VS Code debug and GitHub release process out-of-box.
 - [vite-components-playground](https://xxholly32.github.io/vite-components-playground/) - A common Vue 3 components playground.
+- [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder/) - GitHub template for Electron apps. Used Vite for frontend and backend. Used electron-builder with auto-release and auto-updates for distribution.
```
Comment (innocenzi) [inline README.md:59]: Can you shorten the description a bit? Something like that: 

```suggestion
- [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder/) - GitHub template for Electron apps, with automatic releases and updates thanks to `electron-builder`.
```

**Review by innocenzi — CHANGES_REQUESTED** [review]
I'm not sure I understand this PR's title, isn't `vite-components-playground` the item above your new item, `vite-electron-builder`?

**Comment by cawa-93** [comment]
> I'm not sure I understand this PR's title, isn't `vite-components-playground` the item above your new item, `vite-electron-builder`?

You are right. Forgive my inattention

File: README.md (hunk 1 above)
Comment (cawa-93) [reply README.md:59]: Is it possible to leave "Used Vite for frontend and backend"? In my opinion, this is exactly what makes my template different. While all the alternatives that I've seen use Vite only to build front and backend is built by rollup in my template Vite is used to build all parts of the application.

Comment (innocenzi) [reply README.md:59]: How about this, then: 
> GitHub template for Electron apps using Vite for both back and front-end, with automatic releases and updates.

**Review by innocenzi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vue-vben-admin-2.0](https://github.com/anncwb/vue-vben-admin) - Starter template with Vue 3, TypeScript, Tailwind CSS, Ant Design Vue2.
 - [electron-vue-next](https://github.com/ci010/electron-vue-next) - Starter template using Vue 3 in Vite and latest Electron with fluent VS Code debug and GitHub release process out-of-box.
 - [vite-components-playground](https://xxholly32.github.io/vite-components-playground/) - A common Vue 3 components playground.
-
+- [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder/) - GitHub template for Electron apps using Vite for both back and front-end, with automatic releases and updates.
```
Comment (antfu) [inline README.md:59]:
```suggestion
- [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder/) - Template for Electron apps using Vite for both back and front-end, with automatic releases and updates.

```

**Pull request merged**